### ISSUE TITLE
Bi-weekly meetings for ZEPs

### ISSUE THREAD

**Issue by MSanKeys963** [zarr-developers/zarr-specs#156]
Hi everyone.

I hope you're doing well.

As discussed in the last community call, we're planning to hold bi-weekly meetings to discuss, resolve outstanding issues and accept/reject the ZEPs. Currently, there are two open ZEPs for review and feedback: [ZEP1](https://zarr.dev/zeps/draft/ZEP0001.html) and [ZEP2](https://zarr.dev/zeps/draft/ZEP0002.html).

The PR to the `zarr-specs` repo for ZEP1 & ZEP2 can be seen [here](https://github.com/zarr-developers/zarr-specs/pull/149) and [here](https://github.com/zarr-developers/zarr-specs/pull/152), respectively. Please review the narrative document on the [ZEP website](https://zarr.dev/zeps/) and the PR before joining the ZEP meetings.

This issue is to decide a time suitable for everyone so we can meet and collectively work on the ZEPs.

Here are the timings; please vote by reacting with an emoji:

- 👍🏻: Tuesdays @ `21:30 IST/18:00 CEST/17:00 BST/12:00 EDT/09:00 PDT`
- 😄: Wednesdays @ `21:30 IST/18:00 CEST/17:00 BST/12:00 EDT/09:00 PDT`
- 🎉: Thursdays @ `21:30 IST/18:00 CEST/17:00 BST/12:00 EDT/09:00 PDT`
- ❤️: Fridays @ `21:30 IST/18:00 CEST/17:00 BST/12:00 EDT/09:00 PDT`
- 🚀: Saturdays @ `21:30 IST/18:00 CEST/17:00 BST/12:00 EDT/09:00 PDT`

I understand these timings may not be favourable to everyone, if you have alternate timings, please let me know in the comments, and we'll try to work it out.

Please note that we aim to have these calls in the same week as our bi-weekly community calls. For more info, please check the calendar here: https://zarr.dev/community-calls/

Cc: @zarr-developers/implementation-council @zarr-developers/steering-council @jstriebel @normanrz @alimanfoo @martindurrant @DennisHeimbigner @ivirshup

Please tag/mention if I missed anyone. Thanks!

**Comment by jakirkham** [comment]
Above we list weekdays. Are those the same weekdays in all timezones that are listed? Or are some weekdays different in other timezones? And if the latter, what timezone(s) are the weekdays based off of?

**Comment by MSanKeys963** [comment]
It's the same weekday in every timezone. For, e.g. when I join the community call, it's 23:30 for me on a Wednesday night and 20:00 CEST/19:00 BST/14:00 EDT/11:00 PDT in other timezones.

**Comment by jakirkham** [comment]
Thanks for confirming Sanket 🙏

Second question is what timezone will we pin this to? Asking as DST is still a thing and it will be ending in a few months.

**Comment by MSanKeys963** [comment]
I'd prefer to go with [IST](https://en.wikipedia.org/wiki/Indian_Standard_Time) as Indian Standard Time doesn't follow DST, and it'll be easier for everyone.

**Comment by jakirkham** [comment]
Sorry, but would push back on the notion that not using DST _necessarily_ makes things easier. Have one meeting like this in UTC and it swings around every time DST is entered or left, which results in schedule conflicts and folks sometimes missing the meeting as a result.

We can of course choose (as a group) whether or not to use DST. Though would suggest we pick something that works for most people attending.

**Comment by jbms** [comment]
Perhaps we can just pick a time that works for now --- I, at least, can't commit to my availability months in the future anyway.

**Comment by MSanKeys963** [comment]
Are you mentioning the ZSC Bi-Weekly calls @jakirkham?

**Comment by jakirkham** [comment]
Right, but it shouldn't present an issue if the two are on alternating weeks

**Comment by MSanKeys963** [comment]
That's what I'm aiming for.

**Comment by MSanKeys963** [comment]
Thanks for your votes and input, everyone. I've scheduled the ZEPs meetings on Thursdays every two weeks.

More info here: https://zarr.dev/zeps/meetings/
